monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/VennDiagram/index.js

```diff
@@ -0,0 +1,369 @@
+import React from 'react'
```

Comment:
Please separate this file into 2 files: `ResponsiveVennDiagram.js` and `VennDiagram.js`

Reply:
Also rename the CSS to VennDiagram.css

Reply:
Done